client/app/styles/_variables.scss

```diff
@@ -58,5 +58,5 @@ $search-border-width: 1px;
 $timeline-height: 55px;
 
 /* specific elements */
-$body-background-color: linear-gradient(30deg, $background-color 0%, $background-lighter-color 100%);
+$body-background-color: $background-average-color;
```

Comment:
Finally! :)

btw. there's this whole mix of finely graded shades of background color defined and they all seem to be used somewhere - hopefully we'll be able to remove some as we keep unifying our color language:

```
$background-color: lighten($primary-color, 66%);
$background-lighter-color: lighten($background-color, 8%);
$background-average-color: mix($background-color, $background-lighter-color);
$background-darker-color: darken($background-color, 8%);
$background-darker-secondary-color: darken($background-color, 4%);
$background-dark-color: $primary-color;
```

Reply:
Yes, we'll get rid of a lot of those as time goes on. :)